plugins/cms-export/src/csv.ts

```diff
@@ -69,6 +70,16 @@ export function getDataForCSV(fields: CollectionField[], items: CollectionItem[]
                     continue
                 }
 
+                case "multiCollectionReference": {
+                    if (!Array.isArray(value) || value.some(v => typeof v !== "string")) {
+                        columns.push("")
+                        continue
+                    }
+
+                    columns.push(`${value.join(",")}`)
```

Comment:
Can we add an extra space after the comma?

Reply:
I think that would be incorrect. Standard way to add an array of strings to a csv would be to quote and comma-join.

Reply:
Okay. I didn't know 👌